contracts/src/Surge.sol

```diff
@@ -205,14 +137,16 @@ contract Surge is ERC721, ReentrancyGuard, Ownable, ERC721Enumerable {
         presaleIsActive = false;
     }
 
-    function getTokensMinted() public view returns (uint256) {
-        return _tokenIds.current() + 1;
+    function setMerkleRoot(bytes32 _merkleRoot) public onlyOwner {
+        merkleRoot = _merkleRoot;
     }
 
-    function withdrawAll() public payable onlyOwner {
-        require(payable(msg.sender).send(address(this).balance));
+    function withdrawAll() public payable onlyOwner nonReentrant {
```

Comment:
What is this for?

Reply:
to withdraw all funds from the contract if needed.